impl/src/main/java/com/okta/sdk/impl/config/ClientConfiguration.java

```diff
@@ -49,6 +51,11 @@
     private RequestAuthenticatorFactory requestAuthenticatorFactory = new DefaultRequestAuthenticatorFactory();
     private AuthenticationScheme authenticationScheme;
     private BaseUrlResolver baseUrlResolver;
+    private String authorizationMode;
+    private String clientId;
+    private List<String> scopes = new ArrayList<>();
```

Comment:
nit, we might want to make this null, in order to see if the property was set or not (there are other ways to do that too, so your call)

Reply:
will do

Reply:
I'm not against leaving it like this either, as long as the initial config parseing works an empty array is likely easier to deal with than null :)

Reply:
Yeah, either approach is fine.